zbus/src/fdo/dbus.rs

```diff
@@ -105,6 +105,31 @@ pub enum ReleaseNameReply {
     NotOwner = 0x03,
 }
 
+/// The return code of the [`start_service_by_name`] method.
+///
+/// [`start_service_by_name`]: struct.DBusProxy.html#method.start_service_by_name
```

Comment:
* Let's add a note here why this is not yet the return type of `start_service_by_name` and that it will be in 6.0.
* Let's make use of the rustdoc feature to link and use the full path:

```suggestion
/// The return code of the [`DBusProxy::start_service_by_name`] method.
```